murdock-worker/murdock-slave-init.sh

```diff
@@ -44,7 +45,9 @@ _start() {
         ${MURDOCK_DOCKER_ARGS} \
         -e CCACHE="ccache" \
         -e CCACHE_MAXSIZE \
+        -e CCACHE_MAXFILES \
         -e DWQ_SSH \
+        -e JOBS="${MURDOCK_JOBS}" \
```

Comment:
I'm using locally a hardcoded `-e JOBS=6`, please double checking that I didn't fuck up shell syntax when making this parametric.